src/parcsr_ls/par_amgdd_helpers.c

```diff
@@ -7,7 +7,7 @@
 
 #include "_hypre_parcsr_ls.h"
 #include "_hypre_utilities.h"
-#include "hypre_hopscotch_hash.h"
+#include "hopscotch_hash.h"
```

Comment:
This needs to be added to `_hypre_utilities.h` and removed from here and elsewhere in hypre.